packages/google-cloud-bundle/composer.json

```diff
@@ -18,7 +18,10 @@
     "require": {
         "php": "^7.1",
         "google/cloud-storage": "^1.10",
-        "shopsys/framework": "dev-master",
+        "shopsys/form-types-bundle": "@dev",
```

Comment:
change it to dev-master for all shopsys/* because of `utils/releaser` package

Reply:
Why? What would happen using `@dev`?

According to [Composer docs](https://getcomposer.org/doc/04-schema.md#package-links), this is the correct solution (as it's not a dependency, rather a allowed expection to the minimal-stability rule).

Reply:
@PetrHeinz `@dev` is really just allowing to install transitive `dev-*` dependencies.

But this package depends on `shopsys/framework`, so the `shopsys/framework` must be required in `dev-master` dependency (as we are not allowing installation, but it is requirement).

What do you think?

Reply:
Well, there is a service definition that decorates `Shopsys\FrameworkBundle\Component\Filesystem\MainFilesystemFactory` in `services.yml`, which is a dependency on `shopsys/framework`. I agree :+1:

Reply:
This conversation is closed by https://github.com/shopsys/shopsys/pull/878